src/store/authentication-store.js

```diff
@@ -1,6 +1,8 @@
 import { Store } from 'consus-flux';
 
+/* eslint-disable no-unused-vars */
```

Comment:
This is fine for now since this is basically a stubbed class, but let's try to avoid disabling the linter.